tasks-by-users/Rehan-Khan-Engineer/01-e-commerce-website/contact.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>RK-Accessories</title>
+    <link href="contact.css" rel="stylesheet" />
+  </head>
+  <body>
+    <div class="header">
+      <h1>Get In Touch With Us</h1>
+
+      <p>Want to conncet with us? Here is how you can reach us.</p>
+    </div>
+    <div class="row">
+      <div class="col-3-1">
+        <img src="ferguson images/email-us-1805514_960_720.webp" />
+        <h5><a href="">rk116159@gmail.com</a></h5>
+      </div>
+      <div class="col-3-1">
+        <img src="ferguson images/call-us-1817506_960_720.webp" />
+        <h5><a href="">+91 8655231127</a></h5>
+      </div>
+      <div class="col-3-3">
+        <img src="ferguson images/linked-in-2668692_960_720.webp" />
+        <h5>
+          <a
+            href="https://www.linkedin.com/in/rehan-khan-engineer/"
+            target="_blank"
```

Comment:
What will this attribute do ? @Rehan-Khan-Engineer

Reply:
_self
 @Abbas4tech